### ISSUE TITLE
[A11y] Heading is not identified as a heading by screen reader

### ISSUE THREAD

**Issue by hannahqin** [microsoft/vscode#94955]
Version: 1.44.0
Commit: 2aae1f26c72891c399f860409176fe435a154b13
Date: 2020-04-08T08:23:56.137Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0

**Steps to Reproduce:**
1. Go to Extensions
2. Choose any extension to view its details.
3. Launch VoiceOver and navigate to the extension heading (e.g. "Python").

**Expectation:** 
"Python" should be announced as a heading so that screenreader users are aware of its context on the page.

**Reality:**
Python is announced as a "text element."

![A11y_VSCode_Sreenreader_heading](https://user-images.githubusercontent.com/9788775/79025044-8dc3e780-7b39-11ea-85c2-941001b032d4.gif)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@sandy081 maybe we can tackle this as part of https://github.com/microsoft/vscode/issues/88792

A very easy fix would be to just use `role: heading`
More details here https://w3c.github.io/aria/#heading

Leaving up to @sandy081, if you do not have time maybe provide some code pointers so users might takle this via a PR. Thanks!

**Comment by sandy081** [comment]
Added heading role to extension name element
